manual/biomes.rst

```diff
@@ -0,0 +1,79 @@
+Biomes, Temperature, and Humidity
+=================================
+
+Worldengine uses a complex system to model the environment. For the ease of use for beginners Worldengine is setup to default to a planet with an ecological balance similar to Earth's.
+
+While the system is technically complex, in that it has multiple integrated parts, it is not terribly difficult to understand. In this document we will first explain the general model used for placing biomes and then explain how the user can alter different variables to produce non-terrestrial planets.
+
+Biomes
+------
+
+Worldengine implements a version of the `Holdridge life zones <https://en.wikipedia.org/wiki/Holdridge_life_zones>`_ model for the placement of biomes. While the standard Holdridge life zone model uses three axes Worldengine only concerns itself with the temperature and precipitation axes.
+
+.. image:: https://raw.githubusercontent.com/Mindwerks/worldengine-data/master/docs/Holdridge.png
+   :align: center
+   :width: 95%
+
+As an example, an area of the planet that is **Boreal** in temperature and **Semiarid** in humidity will be classified as **Boreal Wet Forest**. 
+
+While many people regard the |Koppen|_  system as superior it requires the ability to predict seasonal weather. This would require a highly complicated weather simulation system which Worldengine does not currently have. As a result all Temperature and Humidity values are given in terms of yearly averages.
+
+
+.. |Koppen| replace:: K |o| ppen climate classification
+.. _Koppen: https://en.wikipedia.org/wiki/K%C3%B6ppen_climate_classification
+
+.. |o| unicode:: 0xf6
```

Comment:
I tried using unicode initially and somewhere along the line the character was getting nuked, so I think this is a more reliable method. I'll add the comment to make it clearer, however.